Gemfile

```diff
@@ -24,7 +24,7 @@ gem "curb", "~> 0.8"
 
 gem "nokogiri", "~> 1.6"
 
-gem "rails_12factor"
+gem "rails_12factor", group: :production
```

Comment:
yeah, if we are following the http://12factor.net/config we should stick with the 3 `enviroments` from Rails, (production, development and tests) and store the config in environment variables so we can have multiple apps with the rails production enviroment.